team-manager/src/main/java/ua/knu/knudev/teammanager/service/DepartmentService.java

```diff
@@ -0,0 +1,45 @@
+package ua.knu.knudev.teammanager.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ua.knu.knudev.teammanager.domain.Department;
+import ua.knu.knudev.teammanager.repository.DepartmentRepository;
+import ua.knu.knudev.teammanagerapi.dto.AcademicUnitsIds;
+import ua.knu.knudev.teammanagerapi.exception.DepartmentException;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class DepartmentService {
+    private final DepartmentRepository departmentRepository;
+
+    public Department getById(UUID id) {
+        return departmentRepository.findById(id)
```

Comment:
please not in return it is hard to debug case

Reply:
done